src/components/dragons/Dragon.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  fetchDragons, selectAllDragons, bookDragons,
+} from '../../redux/Dragons/dragonSlice';
+import './dragon.css';
+
+function Dragon() {
+  const dispatch = useDispatch();
+  const dragons = useSelector(selectAllDragons);
+  useEffect(() => {
+    dispatch(fetchDragons());
+  }, [dispatch]);
+
```

Comment:
- [ ] Great job implementing the project requirements, however please notice that when I book a rocket or a dragon, and go to my profile it is there, but when I come back to the rocket or dragons page the reserved badge is gone it is also not present under my profile section, I have recorded a small video to make it clear :point_down: 

https://user-images.githubusercontent.com/97521557/225404090-e5c2290a-93a2-4ba7-931d-042903b9e1d0.mp4

Please try to fix this by adding a conditional check like you have done for the missions, in the video you will see the mission when joined is preserved when I visit the mission page or my profile section again.